tests/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
-
-include(${CMAKE_SOURCE_DIR}/cmake/common_build_flags.cmake)
+include(${PROJECT_SOURCE_DIR}/cmake/common_build_flags.cmake)

```

Comment:
`PROJECT_SOURCE_DIR` is required when running through ExternalProject or FetchContent.